tck/src/main/java/org/eclipse/microprofile/config/tck/ImplicitConverterTest.java

```diff
@@ -78,6 +79,14 @@ public void testImplicitConverterStringValueOf() {
         Assert.assertNotNull(value);
         Assert.assertEquals(value.getVal(), "stringValueOf");
     }
+    
+    @Test
+    public void testImplicitConverterCharSequenceParse() {
+        ConvTestTypeWCharSequenceParse value = config.getValue("tck.config.test.javaconfig.converter.implicit.charSequenceParse", 
```

Comment:
let's use an actual class (e.g. java.time.Year). We don't need to create a weirdly named class to check that implicit converter for Java runtime classes work

Reply:
added more tests to cover your usage.